src/Command/Tunnel/TunnelOpenCommand.php

```diff
@@ -0,0 +1,198 @@
+<?php
+namespace Platformsh\Cli\Command\Tunnel;
+
+use Platformsh\Cli\Command\CommandBase;
+use Platformsh\Cli\Util\PortUtil;
+use Platformsh\Cli\Util\ProcessUtil;
+use Platformsh\Cli\Util\RelationshipsUtil;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\StreamOutput;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Process\ProcessBuilder;
+
+class TunnelOpenCommand extends CommandBase
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('tunnel:open')
+            ->setDescription("Open SSH tunnel(s) to the app's services");
+        $this->addProjectOption();
+        $this->addEnvironmentOption();
+        $this->addAppOption();
+    }
+
+    public function isEnabled()
+    {
+        return ProcessUtil::supported() && parent::isEnabled();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->validateInput($input);
+        $project = $this->getSelectedProject();
+        $environment = $this->getSelectedEnvironment();
+        $appName = $this->selectApp($input);
+
+        $sshUrl = $environment->getSshUrl($appName);
+
+        $util = new RelationshipsUtil($this->stdErr);
+        $relationships = $util->getRelationships($sshUrl);
+        if (!$relationships) {
+            $this->stdErr->writeln('No app relationships found.');
+            return 1;
+        }
+
+        $tunnelDir = $this->getConfigDir() . '/tunnel';
+        if (!is_dir($tunnelDir)) {
+            mkdir($tunnelDir, 0700, true);
+        }
+
+        $logFile = $tunnelDir . '/log.txt';
+        $log = new StreamOutput(fopen($logFile, 'a'));
+
+        ProcessUtil::fork();
+
+        /** @var Process[] $processes */
+        $processes = [];
+        $error = false;
+        foreach ($relationships as $name => $services) {
+            foreach ($services as $key => $service) {
+                $remoteHost = $service['host'];
+                $remotePort = $service['port'];
+
+                $start = isset($localPort) ? $localPort + 1 : 30000;
+                $localPort = PortUtil::getPort($start);
+                $tunnelInfo = [
+                    'projectId' => $project->id,
+                    'environmentId' => $environment->id,
+                    'appName' => $appName,
+                    'relationship' => $name,
+                    'key' => $key,
+                    'remotePort' => $remotePort,
```

Comment:
I'm going to write all that info to a file, so you can have a `tunnel:status` command or similar to show all the tunnels you opened